tests/simpletest.py

```diff
@@ -227,6 +227,37 @@ def test_copy_template(self):
             bf2.union(self.bf)  # Asserts copied bloom filter is comparable
             self._check_filter_contents(bf2)
 
+    def test_create_with_hash_seeds(self):
+        cust_seeds = [getrandbits(32) for i in range(30)]
+        bf = pybloomfilter.BloomFilter(self.FILTER_SIZE,
+                                       self.FILTER_ERROR_RATE,
+                                       self.tempfile.name,
+                                       hash_seeds=cust_seeds)
+        bf_seeds = bf.hash_seeds.tolist()
+        self.assertEqual(cust_seeds, bf_seeds)
+
+    def test_create_with_hash_seeds_and_compare(self):
+        test_data = "test"
+        bf1 = pybloomfilter.BloomFilter(self.FILTER_SIZE,
+                                        self.FILTER_ERROR_RATE,
+                                        self.tempfile.name)
+        bf1.add(test_data)
+        bf1_seeds = bf1.hash_seeds.tolist()
+        bf1_base64 = bf1.to_base64()
+        bf1.close()
+
+        bf2 = pybloomfilter.BloomFilter(self.FILTER_SIZE,
+                                        self.FILTER_ERROR_RATE,
+                                        self.tempfile.name,
+                                        hash_seeds=bf1_seeds)
+        bf2.add(test_data)
+        bf2_seeds = bf2.hash_seeds.tolist()
+        bf2_base64 = bf2.to_base64()
+        bf2.close()
```

Comment:
[todo] we should add `__enter__` and `__exit__` methods to the `BloomFilter` class. So we can use the `context manager` patterns.

This would let us do:

```py
with pybloomfilter.BloomFilter(...) as bf:
  bf.add(test_data)
```

Reply:
Nice feature. Can work on this, but I will probably issue a separate PR for this.

Would also implicitly allow having something like
```
with pybloomfilter.BloomFilter.from_base64(...) as bf:
```
as well as
```
with pybloomfilter.BloomFilter.open(...) as bf:
```

Reply:
Sounds great!